### DISCUSSION TITLE
2011/12/12/%E7%94%A8%E9%BC%A0%E6%A0%87%E5%9C%A8%E6%B3%A2%E5%BD%A2%E5%9B%BE%E6%8E%A7%E4%BB%B6%E4%B8%8A%E9%80%89%E5%8F%96%E4%B8%80%E6%9D%A1%E6%9B%B2%E7%BA%BF

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#349] [Announcements]
# 2011/12/12/%E7%94%A8%E9%BC%A0%E6%A0%87%E5%9C%A8%E6%B3%A2%E5%BD%A2%E5%9B%BE%E6%8E%A7%E4%BB%B6%E4%B8%8A%E9%80%89%E5%8F%96%E4%B8%80%E6%9D%A1%E6%9B%B2%E7%BA%BF

https://labview.qizhen.xyz/

https://blog.qizhen.xyz/2011/12/12/%E7%94%A8%E9%BC%A0%E6%A0%87%E5%9C%A8%E6%B3%A2%E5%BD%A2%E5%9B%BE%E6%8E%A7%E4%BB%B6%E4%B8%8A%E9%80%89%E5%8F%96%E4%B8%80%E6%9D%A1%E6%9B%B2%E7%BA%BF

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Jerry
2011年12月14日 15:19 编辑
老师，你的Waveform怎么这么好看，怎么做出来的？

回复
ruanqizhen
2011年12月15日 09:48 编辑
这是LabVIEW2011自带的控件。2011改进了界面，好看多了。

回复
